pipelines/build/prTester/pr_test_pipeline.groovy

```diff
@@ -121,18 +121,7 @@ class PullRequestTestPipeline implements Serializable {
 }
 
 Map<String, ?> defaultTestConfigurations = [
-    "x64Linux": [
-        "hotspot",
-        "openj9"
-    ],
-    "aarch64Linux": [
-        "hotspot",
-        "openj9"
-    ],
-    "x64Windows": [
-        "hotspot"
-    ],
-    "x64Mac": [
+    "x32Windows": [
```

Comment:
Not sure about this change - why remove the others and only have win32?

Reply:
this is clearly for testing, needs to be reverted. The PR is in draft form right now though

Reply:
Yes it's for testing purposes only to prevent the pr tester from running everything. I'll remove the 2nd commit (the one that removes the platforms) before I move this out of draft

Reply:
Commit has been removed. @gdams @karianna Could I get reviews again?